lib/Echidna/Campaign.hs

```diff
@@ -166,9 +174,38 @@ evalSeq v e = go [] where
     case xs of []     -> pure []
                (y:ys) -> e y >>= \a -> ((y, a) :) <$> go (y:r) ys
 
+-- | Given a call sequence that produces Tx with gas >= g for f, try to randomly generate
+-- a smaller one that achieves at least that gas usage
+shrinkGasSeq :: ( MonadRandom m, MonadReader x m, MonadThrow m
```

Comment:
This one is unused, right?

Reply:
Yes, that's for the future gas-based shrinking